src/fromager/wheels.py

```diff
@@ -94,6 +98,11 @@ def build_wheel(
 
     builder(ctx, build_env, extra_environ, req, sdist_root_dir)
     wheels = list(ctx.wheels_build.glob("*.whl"))
+
+    # End the timer
+    end = datetime.now().replace(microsecond=0)
+    logger.info(f"Time taken to build wheel for {req.name}: {end - start}")
```

Comment:
To keep the logging format consistent start log statements with `req.name` like here - 
https://github.com/python-wheel-build/fromager/pull/192/files#diff-e7392f8e6edefea9b61e882f028ac83e81d7c40ca569de4fb02b380ffe2359f2L78